### ISSUE TITLE
CI failure: broken link(s) to source files?

### ISSUE THREAD

**Issue by bjarchi** [MarlinFirmware/MarlinDocumentation#108]
Travis CI has been failing on recent documentation PRs, with what appear to be broken links to source file(s). I tried looking into this but I'm not familiar enough with how Jekyll and Github interact to fix this. 

Failing section of CI run:
```bash
/home/travis/build/MarlinFirmware/MarlinDocumentation/_site/docs/development/lcd_language.html
  *  External link https://github.com/MarlinFirmware/Marlin/blob/Development/Marlin/ultralcd_implementation_hitachi_HD44780.h failed: 404 No error
  *  internally linking to /MarlinFirmware/Marlin/tree/1.1.x/buildroot/share/fonts#readme, which does not exist (line 46)
     <a href="/MarlinFirmware/Marlin/tree/1.1.x/buildroot/share/fonts#readme"> file</a>
  *  trying to find hash of /MarlinFirmware/Marlin/tree/1.1.x/buildroot/share/fonts#readme, but /home/travis/build/MarlinFirmware/MarlinDocumentation/_site/MarlinFirmware/Marlin/tree/1.1.x/buildroot/share/fonts does not exist (line 46)
     <a href="/MarlinFirmware/Marlin/tree/1.1.x/buildroot/share/fonts#readme"> file</a>
htmlproofer 3.7.4 | Error:  HTML-Proofer found 3 failures!
```

I think the first failure may be cause by `ultralcd_implementation_hitachi_HD44780.h` being renamed to `ultralcd_impl_hitachi_HD44780.h`. I'm not sure about the second error, since `MarlinFirmware/Marlin/tree/1.1.x/buildroot/share/fonts#readme` works in my browser. 

I'll submit a PR if I can figure out what's going on, otherwise I wanted to document it.

**Comment by bjarchi** [comment]
Looks like this was easier than I expected - I think it's fixed in PR #106 (CI passes; I think the links are still as intended.)

**Comment by bjarchi** [comment]
Fixed w/ PR #106 - merged - so issue closed.
